diff --git a/Makefile b/Makefile
--- Makefile
+++ Makefile
@@ -4,26 +4,9 @@
 # this is the upstream libcurl-impersonate version
 VERSION := 0.7.0
 CURL_VERSION := curl-8_7_1
 
-$(CURL_VERSION):
-	curl -L https://github.com/curl/curl/archive/$(CURL_VERSION).zip -o curl.zip
-	unzip -q -o curl.zip
-	mv curl-$(CURL_VERSION) $(CURL_VERSION)
-
-curl-impersonate-$(VERSION)/chrome/patches: $(CURL_VERSION)
-	curl -L "https://github.com/yifeikong/curl-impersonate/archive/refs/tags/v$(VERSION).tar.gz" \
-		-o "curl-impersonate-$(VERSION).tar.gz"
-	tar -xf curl-impersonate-$(VERSION).tar.gz
-
-.preprocessed: curl-impersonate-$(VERSION)/chrome/patches
-	cd $(CURL_VERSION)
-	for p in $</curl-*.patch; do patch -p1 < ../$$p; done
-	# Re-generate the configure script
-	autoreconf -fi
-	mkdir -p ../include/curl
-	cp -R include/curl/* ../include/curl/
-	# Sentinel files: https://tech.davis-hansson.com/p/make/
+.preprocessed:
 	touch .preprocessed
 
 local-curl: $(CURL_VERSION)
 	cp /usr/local/lib/libcurl-impersonate-chrome* /Users/runner/work/_temp/install/lib/

